mongodbatlas/logs_test.go

```diff
@@ -0,0 +1,25 @@
+package mongodbatlas
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+)
+
+func TestLogs_Get(t *testing.T) {
+	setup()
+	defer teardown()
+
+	groupID := "1"
+	cluster := "test-username"
+	log := "log"
+
+	mux.HandleFunc(fmt.Sprintf("/groups/%s/clusters/%s/logs/%s", groupID, cluster, log), func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, http.MethodGet)
```

Comment:
[q] can you make this return a random text?
then you can do something like
```
buf := new(bytes.Buffer)
client.Logs.Get(ctx, groupID, cluster, log, buf, nil)
if buf.String() != "random string" {
 t.Error....
}
```
(advantages of depending on an interface ;))